docs/REST_API-Draft/draft.md

```diff
@@ -35,145 +35,178 @@ Statusy odpowiedzi:
 ### Wylogowanie
```

Comment:
Rozbij to jak możesz na 2 pliki (Autoryzacja i Operacje na danych oddzielnie).
Dodatkowo brakuje opisu modeli do komunikacji po TCP/IP Sensor -> Monitor